pkg/probe/http/http.go

```diff
@@ -129,7 +129,8 @@ func DoHTTPProbe(url *url.URL, headers http.Header, client GetHTTPInterface) (pr
 	if res.StatusCode >= http.StatusOK && res.StatusCode < http.StatusBadRequest {
 		if res.StatusCode >= http.StatusMultipleChoices { // Redirect
 			klog.V(4).Infof("Probe terminated redirects for %s, Response: %v", url.String(), *res)
-			return probe.Warning, body, nil
+			// redirect is terminated, so ignore the body
+			return probe.Warning, "Probe terminated redirects", nil
```

Comment:
I wonder if anybody may have a dependency on old behavior and will expect to see a redirect link that often included into the response body as part of the warning. This is not a documented feature, but I don't think breaking this behavior is OK.

What do you think of using this text only when body is empty?

Reply:
Do you mean to change it to 
```suggestion
			if body == nil {
				return probe.Warning, "Probe terminated redirects", nil
			}
			return probe.Warning, body, nil
```

Reply:
Yes. Maybe check for empty as well like `if len(body) > 0 {`

Reply:
Done